internal/server/v2/resolve/golden/resolve_description/abbreviated_names.json

```diff
@@ -0,0 +1,37 @@
+{
+  "entities": [
+    {
+      "node": "California, USA",
+      "resolvedIds": [
+        "geoId/06",
+        "geoId/2111872",
+        "geoId/2412150",
+        "geoId/2910468",
+        "geoId/4210768"
+      ]
+    },
+    {
+      "node": "ME",
+      "resolvedIds": [
+        "country/MNE",
```

Comment:
To confirm, why is the order here country before state, while in recognize places, state before country?

If we had to pick, I might slightly go with the state before country order...

Reply:
I think RecognizePlaces sort places by population. 
However, ResolveDescription (which calls BulkFindEntities) doesn't sort the resolved entities. If needed, we can also sort (maybe in a future PR) by population, but I'm not sure if that's necessary, because ResolveDescription can also take placeType as an input to filter, so if the user really wants "State" or "Country", he/she can specify it in the query, like "<-description{typeOf:State}->dcid"